backend/src/routes/aggregators/applicationsOCP.ts

```diff
@@ -265,12 +266,19 @@ function getValues(labels: { annotation: any; value: any }[]) {
   }
 }
 
+export const systemAppNamespacePrefixes: string[] = []
+export async function discoverSystemAppNamespacePrefixes() {
+  if (!systemAppNamespacePrefixes.length) {
+    systemAppNamespacePrefixes.push('openshift')
+    systemAppNamespacePrefixes.push('hive')
+    systemAppNamespacePrefixes.push('rhacm')
+    systemAppNamespacePrefixes.push('open-cluster-management')
```

Comment:
Sorry John - by "RHACM installation namespace" I still meant you should look up what that actually is... not the hardcoded `rhacm` namespace that I used as an example.
```suggestion
    systemAppNamespacePrefixes.push('open-cluster-management')
    const mch = await getMultiClusterHub()
    if (mch?.metadata?.namespace && mch.metadata.namespace !== 'open-cluster-management') {
      systemAppNamespacePrefixes.push(mch.metadata.namespace )
    }
```

Reply:
I got to remember to see if there's a suggested change 🥇